Rakefile

```diff
@@ -36,4 +36,9 @@ task lint: %i[rubocop environment] do
   sh "yarn run lint"
 end
 
-task default: [:lint, :spec, "app:jasmine:ci"]
+desc "Jasmine"
+task :jasmine do
+  sh "yarn run jasmine:ci"
+end
```

Comment:
In our Rails applications, I'm going to suggest we define this task in its own file at: `lib/tasks/jasmine.rake`. This would make it consistent with similar tasks, such as [`lint.rake` in content-publisher](https://github.com/alphagov/content-publisher/blob/96c51d89ae3eea8a43b2276a9443b08599d0e8eb/lib/tasks/lint.rake).

This repo is a bit weird since it's both a gem and a Rails app, so I'll leave it up to you to decide which location you think is best for the `:jasmine` task to be defined.

Reply:
FAO @kevindew

Reply:
Yeah looks like there's already a precedence here for Rake tasks defined in that file so I think it's ok here, does some good to have separate files for apps.

Reply:
Good point. I hadn't even spotted that the `Rakefile` already contained other tasks 🙂